release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,472 @@
+:_content-type: ASSEMBLY
+[id="release-notes-40"]
+= Red Hat Advanced Cluster Security for Kubernetes 4.0
+include::modules/common-attributes.adoc[]
+:context: release-notes-40
+
+toc::[]
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime stages of the application lifecycle. It deploys in your infrastructure and integrates with your DevOps tools and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+.Release dates
+[options="header"]
+|====
+|{product-title-short} version |Released on
+|`4.0` |3 May 2023
+|====
+
+[id="about-this-release-40"]
+== About this release
+
+{product-title-short} 4.0 includes the following new features, improvements, and updates:
+
+* xref:../release_notes/40-release-notes.adoc#major-release-version-change[Major release version change to 4.0]
+* xref:../release_notes/40-release-notes.adoc#external-postgresql-database-rn40[Installing Central with an external PostgreSQL database (Technology Preview)]
+* xref:../release_notes/40-release-notes.adoc#acs-cloud-release-rn40[{product-title-managed-short} Limited Availability]
+* xref:../release_notes/40-release-notes.adoc#telemetry-rn40[Telemetry data collection in {product-title-managed-short}]
+* xref:../release_notes/40-release-notes.adoc#rhcos-node-scan-rn40[{op-system-first} node host scanning for security vulnerabilities]
+* xref:../release_notes/40-release-notes.adoc#process-listening-endpoints-rn40[Processes listening on endpoints API]
+* xref:../release_notes/40-release-notes.adoc#network-graph-rn40[Network graph 2.0 (Technology Preview) updates]
+* xref:../release_notes/40-release-notes.adoc#fips-compliance-rn40[FIPS compliance]
+* xref:../release_notes/40-release-notes.adoc#token-expiry-alert-rn40[Alert messages sent to Central logs for expiring tokens]
+* xref:../release_notes/40-release-notes.adoc#sensor-resync-rn40[Improvements for Sensor resync (Technology Preview)]
+* xref:../release_notes/40-release-notes.adoc#doc-additions-rn40[Documentation additions]
+
+[IMPORTANT]
+====
+For offline configured installations that have set the `collector.slimMode` option to `false`: the `rhacs-collector-rhel8` image now contains a subset of kernel modules and eBPF probes available in the support package download. If the collector status is `unhealthy` after an upgrade, follow the instructions for xref:../configuration/enable-offline-mode.adoc#download-kernel-support-package_enable-offline-mode[downloading kernel support packages] and then upload them to Central.
```

Comment:
Suggestion: add a link to https://docs.openshift.com/acs/3.74/configuration/enable-offline-mode.html on the `offline configured` text

Reply:
```suggestion
For xref:../configuration/enable-offline-mode.adoc[offline configured] installations that have set the `collector.slimMode` option to `false`: the `rhacs-collector-rhel8` image now contains a subset of kernel modules and eBPF probes available in the support package download. If the collector status is `unhealthy` after an upgrade, follow the instructions for xref:../configuration/enable-offline-mode.adoc#download-kernel-support-package_enable-offline-mode[downloading kernel support packages] and then upload them to Central.
```